api/swagger/server/restapi/resource/swagger.yml

```diff
@@ -726,7 +726,7 @@ paths:
               properties:
                 id:
                   description: Plugin identifier.
-                  type: integer
+                  type: string
```

Comment:
Why changing it to a string ?

Reply:
because its not possible to represent an int64 in json type integer:
`Par exemple, un entier de 64 bits ne peut pas être représenté au format JSON, car JavaScript et JSON acceptent les entiers jusqu'à 2^53. Par conséquent, un entier 64 bits doit être représenté sous forme de chaîne dans les requêtes/réponses JSON`
from:
https://developers.google.com/discovery/v1/type-format?hl=fr#:~:text=Defined%20by%20the%20JSON%20Schema%20spec.,-integer&text=A%2032%2Dbit%20signed%20integer,value%20of%202%2C147%2C483%2C647%20(inclusive).

this said, i will answer, why not string ?